[fix][doc]add design doc

hf200012 opened the pull request:
add design doc

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

hf200012 committed:
[fix][doc]add design doc

add design doc

yiguolei reviewed (dismissed):
LGTM

carlvinhust2012 commented on docs/en/design/Flink doris connector Design.md:
```diff
@@ -0,0 +1,259 @@
+
+public RowData nextRecord(RowData reuse)  {
+		if (!hasNext) {
+            //After reading the data, return null
```
It is recommended to adjust the indentation.

carlvinhust2012 commented on docs/en/design/doris_storage_optimization.md:
````diff
@@ -0,0 +1,235 @@
+                 +----------------+
+```
+
+non -zero data page32467;- 26500;- 229140;-
````
This line is different from the line 68 of 'zh-CN/design/doris_storage_optimization.md'

carlvinhust2012 commented on docs/en/design/doris_storage_optimization.md:
```diff
@@ -0,0 +1,235 @@
+message SegmentFooterPB {
+	optional uint32 version = 2 [default = 1]; // For version compatibility and upgrade use
+	repeated ColumnPB schema = 5; // Schema of columns
+  optional uint64 num_values = 4; // Number of lines saved in the file
```
It is recommended to adjust the indentation.

wangbo committed:
[fix](storage)bloom filter support ColumnDict (#9167)

bloom filter support ColumnDict(#9167)

morningman committed:
[fix] fix docs build bug  (#9293)

After this PR #9272, the `docs/build_help_zip.sh` will run failed.
This PR fix this issue.
But the help module still has some parse problem, I will fix it in next PR.

This CL mainly changes:
1. fix `docs/build_help_zip.sh` error
2. remove `sql-reference-v2` to `sql-reference`
3. modify build extension github action to run `docs/build_help_zip.sh`

HappenLee committed:
[Vectorized](stream-load-vec) Support stream load in vectorized engine (#8709) (#9280)

Implement vectorized stream load.
Added fe configuration option `enable_vectorized_load` to enable vectorized stream load.

    Co-authored-by: tengjp@outlook.com
    Co-authored-by: mrhhsg@gmail.com
    Co-authored-by: minghong.zhou@163.com
    Co-authored-by: HappenLee <happenlee@hotmail.com>
    Co-authored-by: zhoubintao <35688959+zbtzbtzbt@users.noreply.github.com>

xy720 committed:
[refactor][routineload] Remove unused client object from routine load (#9223)

zhengshiJ committed:
[feature](statistics) Statistics derivation.Step 1:ScanNode implement… (#8947)

* [feature](statistics) Statistics derivation.Step 1:ScanNode implementation

Co-authored-by: jianghaochen <jianghaochen@meituan.com>

wangbo committed:
[test](storage)duplicate model storage layer regression-test (#9285)

* duplicate model storage layer regression-test

* use insert values for batch

Co-authored-by: Wang Bo <wangbo36@meituan.com>

morningman committed:
[fix](materialized-view) fix bug that can not create mv for list partitioned table (#9281)

zenoyang committed:
[fix](profile) Short-circuit and del predicate filter rows are not counted on vectorized exec (#9268)

EmmyMiao87 committed:
(Refactor)[Statistics] Fix lock risks in Statistics Job (#9256)

* (Refactor)[Statistics] Fix lock risks in Statistics Job

1. Remove lock nesting between job and task
2. Solve the deadlock problem during job update
3. Avoid printing the log while holding the lock

* Add log

long2ice committed:
[Features]Add dbt doris adapter (#9299)

* Add dbt doris adapter

* Add licence header to each file

* Fix licence header

morningman committed:
[fix](help-doc) fix format of all sql-manual doc (#9306)

zenoyang committed:
[fix](vectorized) Query get wrong result when ColumnDict concurrent predicate eval (#9270)
